orchestration/dagster_orchestration/hca_orchestration/contrib/retry.py

```diff
@@ -0,0 +1,49 @@
+import time
+from typing import Any, Callable, TypeVar
+
+
+class RetryException(Exception):
+    """
+    Represents a failure after some number of retries
+    """
+    pass
+
+
+def is_truthy(result: Any) -> bool:
```

Comment:
type nit: when writing functions that are expected to work for all possible values, use `object` instead of `Any` as the argument type to indicate "all" rather than "whichever"